### ISSUE TITLE
Upgrading homebridge via button does not recognize forks

### ISSUE THREAD

**Issue by patrik72** [oznu/homebridge-config-ui-x#25]
when using a homebridge fork, like https://github.com/NorthernMan54/homebridge, an upgrade will install the original. So a parameter for the fork would be nice.

**Comment by oznu** [comment]
I'll see if I can make this work well.

It's a little more work than just doing an install using `npm install NorthernMan54/homebridge` since I'll also need to check for version updates from the GitHub repo instead of npm.

**Comment by oznu** [comment]
It was pretty simple. Now available in 2.5.0.

```json
{
    "platform": "config",
    "name": "Config",
    "port": 8080,
    "fork": "northernman54/homebridge"
}
```

The `fork` must be defined in `<username>/<repo name>` format.

Caveats:

* Forks must be on GitHub.
* Support for `master` branches only.
* Must have the name `homebridge` in the `package.json`.
* The fork maintainer must bump the version in the `package.json` for homebridge-config-ui-x to be able to notify you that an update is available.
